### PR TITLE
helm dependencies add cert-manager

### PR DESCRIPTION
**Pull request by zyhui98**
### Ⅰ. Describe what this PR did
#109

### CODE REVIEW

**Review by CH3CHO — CHANGES_REQUESTED** [review]
Could you make cert-manager an optional dependency?

**Comment by zyhui98** [comment]
> Could you make cert-manager an optional dependency?

ok, like other compenent, eg prometheus ?

**Comment by CH3CHO** [comment]
> > Could you make cert-manager an optional dependency?
> 
> ok, like other compenent, eg prometheus ?

Right.

**Comment by zyhui98** [comment]
> > > Could you make cert-manager an optional dependency?
> > 
> > 
> > ok, like other compenent, eg prometheus ?
> 
> Right.

fix ok

File: helm/templates/cert-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,1284 @@
+{{- if .Values.o11y.enabled }}
```
Comment (CH3CHO) [inline helm/templates/cert-manager.yaml:1]: cert-manager 不算 o11y 组件吧

**Review by CH3CHO — CHANGES_REQUESTED** [review]

File: helm/templates/cert-manager.yaml (hunk 1 above)
Comment (zyhui98) [reply helm/templates/cert-manager.yaml]: yes, i changed

Comment (zyhui98) [reply helm/templates/cert-manager.yaml]: > cert-manager 不算 o11y 组件吧

好了

**Review by CH3CHO — APPROVED** [review]

**Pull request merged**